Makefile

```
@@ -400,12 +400,16 @@ $(NOBUILD_TARGETS):
 CLEAN_TARGETS = $(addprefix clean_,$(VALID_TARGETS) )
 TARGETS_CLEAN = $(addsuffix _clean,$(VALID_TARGETS) )
 
-## clean             : clean up temporary / machine-generated files
+## clean             : clean one target (TARGET=<name>), or all targets if TARGET is unspecified
 clean:
```

checkmake (0.3.2)

[warning] Target "clean" should be declared PHONY.

(phonydeclared)